e2e/__tests__/selectProjects.test.ts

```diff
@@ -0,0 +1,82 @@
+/**
```

Comment:
could you add some tests here for the warnings? E.g. not matching names, projects missing names etc?

Reply:
It's good that you asked for tests because the warning on names was not working at all.